composer.json

```diff
@@ -15,14 +15,14 @@
   "type": "neos-package",
   "license": "MIT",
   "require": {
-    "php": "^7.0",
+    "php": "^7.1",
```

Comment:
As Neos 5.0 requires PHP 7.2, this could also be set to 7.2

Reply:
I wanted to keep backwards compatibility.
You could argue that PHP 7.1 will be EOL soon too, but I'd keep it for now (but removing the almost 1 year EOL version 7.0).